.github/workflows/build-skip.yml

```diff
@@ -1,5 +1,6 @@
 name: Build SimApp
 # This workflow allows to skip the build step if the PR does not contain any changes to the code
+# See https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
```

Comment:
Just curious why you found it necessary to add these comments?

Reply:
To make it clearer: https://github.com/cosmos/cosmos-sdk/pull/13361#discussion_r979007431